v2/definitions/VideoJobRequest.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  type: 'object',
+  required: ['video', 'fps', 'classes'],
+  properties: {
+    video: {
+      type: 'string',
+      description: 'URL of the video',
+      maxLength: 512,
+    },
+    fps: {
+      type: 'integer',
+      description: 'Number of frames to be classified, per second.',
+      maximum: 2,
+    },
+    classes: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['class'],
+        properties: {
```

Comment:
Are we planning to add `verify` and `boundingBox` to this object later on?

Reply:
Potentially, yeah. Video will probs receive a complete rework soon